src/simmate/apps/clease/workflows/cluster_high_q.py

```diff
@@ -1,10 +1,45 @@
-from simmate.apps.vasp.inputs import PBE_POTCAR_MAPPINGS
+from simmate.apps.vasp.inputs.potcar_mappings import PBE_ELEMENT_MAPPINGS
```

Comment:
the PR I merged yesterday changed this FYI. You'll want to use `from simmate.apps.vasp.inputs import PBE_POTCAR_MAPPINGS` throughout your app now